packages/pyright-internal/src/tests/logger.test.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * logger.test.ts
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ *
+ * Unit tests for logger.
+ */
+
+import * as assert from 'assert';
```

Comment:
Not sure a test is really merited for this functionality. Unit tests are important in cases where a regression is likely and such a regression will negatively impact users. Low-value unit tests consume development cycles, increase CI wait times, and increase maintenance burden.

This one unit test by itself isn't a big deal, but these add up over time. So if you want to keep this unit test, I'm OK with that, but please take care in what unit tests you add in the future.